stor/s3.py

```diff
@@ -765,3 +769,34 @@ def to_url(self):
         """Returns HTTP url for object (virtual host-style)"""
         return u'https://{bucket}.s3.amazonaws.com/{key}'.format(bucket=self.bucket,
                                                                  key=self.resource)
+
+    def restore(self, tier='Bulk', days=10):
+        """Issue a restore command for a single object from glacier.
+
+        Args:
+            tier (str, optional): restore speed (see Glacier docs for details)
+            days (int, optional): number of days to keep data in S3 post-restore.
```

Comment:
add default value

Reply:
did it, kinda duplicative with actual type signature tho